src/sugar3/graphics/popwindow.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (C) 2016 Abhijit Patel
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+from gi.repository import Gtk
+from gi.repository import Gdk
+from gi.repository import GdkX11
+
+from sugar3.graphics import style
+from sugar3.graphics.toolbutton import ToolButton
+from sugar3.graphics.toolbarbox import ToolbarBox
+
+
+class PopWindow(Gtk.Window):
+    """
+    UI interface for activity Pop-up Windows
```

Comment:
Please write some docs for this.  Here is an example of the format:  https://github.com/sugarlabs/sugar-toolkit-gtk3/pull/256/files#diff-74308e345a99a21f5fbfc47f0dbcc070R106